parsons/databases/redshift/redshift.py

```diff
@@ -401,9 +401,12 @@ def copy(self, tbl, table_name, if_exists='fail', max_errors=0, distkey=None,
                 characters.
             statupate: boolean
                 Governs automatic computation and refresh of optimizer statistics at the end
-                of a successful COPY command.
+                of a successful COPY command. If ``True`` explicitly sets ``statupate`` to on, if
+                ``False`` explicitly sets ``statupate`` to off. Defaults to ``None``.
```

Comment:
Please include a note telling the user that  

``If STATUPDATE is used, the current user must be either the table owner or a superuser.``

Reply:
Perhaps explain to folks what happens if it is set to None. Or at least link to the redshift documentation?